### DISCUSSION TITLE
How to change the default tB icon for the built exe?

### DISCUSSION THREAD

**Discussion by haplox** [twinbasic/twinbasic#1381] [Q&A]
Hello,

I know than I can do it with Resource Hacker (for example), but I have tried it from the IDE, but it is impossible neither to erase the default tB icon, nor to add any other personal image. On ly in Resources folder in the root of Project Explorer, but this does not change anything building the EXE.

Thanks in advance

**Comment by WaynePhillipsEA** [comment]
Changing the icon in the Resources folder is all that is needed.   However, Explorer caches the icon, so you need to run the following at the command line in order for Explorer to show the changed icon:

`ie4uinit -show`

**Comment by haplox** [comment]
Thanks @WaynePhillipsEA , it works, but I do not understand why this behaviour
